.madrun.js

```diff
@@ -42,45 +42,79 @@ module.exports = {
     'lint': () => run('lint:progress'),
     'lint:progress': () => run('lint:base', '-f progress'),
     'lint:stream': () => run('lint:base', '-f stream'),
-    
+
     'test:base': () => {
         const cmd = 'tape';
         const names = `'test/**/*.js' '{client,static,common,server}/**/*.spec.js'`;
-        
+
         return `${cmd} ${names}`;
     },
-    
+
     'test': () => `${env} ${run('test:base')}`,
     'test:client': () => `tape 'test/client/**/*.js'`,
     'test:server': () => `tape 'test/**/*.js' 'server/**/*.spec.js' 'common/**/*.spec.js'`,
     'wisdom': () => run(['lint:all', 'build', 'test']),
     'wisdom:type': () => 'bin/release.js',
     'docker:pull:node': () => 'docker pull node',
     'docker:pull:alpine': () => 'docker pull mhart/alpine-node',
-    'docker:push': () => `docker push coderaiser/cloudcmd:${version}`,
-    'docker:push:latest': () => 'docker push coderaiser/cloudcmd:latest',
+    'docker:pull:arm32': () => 'docker pull arm32v7/node:slim',
+    'docker:pull:arm64': () => 'docker pull arm64v8/node:slim',
+    'docker:push': () => `docker push coderaiser/cloudcmd:${version}-x64`,
+    'docker:push:latest': () => 'docker push coderaiser/cloudcmd:latest-x64',
     'docker:push:alpine': () => `docker push coderaiser/cloudcmd:${version}-alpine`,
     'docker:push:alpine:latest': () => 'docker push coderaiser/cloudcmd:latest-alpine',
-    'docker:build': () => `docker build -t coderaiser/cloudcmd:${version} .`,
-    'docker:build:alpine': () => `docker build -f Dockerfile.alpine -t coderaiser/cloudcmd:${version}-alpine .`,
+    'docker:push:arm32': () => `docker push coderaiser/cloudcmd:${version}-arm32`,
+    'docker:push:arm32:latest': () => 'docker push coderaiser/cloudcmd:latest-arm32',
+    'docker:push:arm64': () => `docker push coderaiser/cloudcmd:${version}-arm64`,
+    'docker:push:arm64:latest': () => 'docker push coderaiser/cloudcmd:latest-arm64',
+    'docker:build': () => `docker build -f docker/Dockerfile -t coderaiser/cloudcmd:${version}-x64 .`,
```

Comment:
Could we create functions in the bottom for each steps to avoid duplication :)?

```js
function dockerPush(type, version = 'latest') {
    return `docker push coderaiser/cloudcmd:${version}-${type}`;
}
```

and use as:

```js
module.exports = {
    'docker:push:arm32:latest': dockerPush('arm32'),
    'docker:push:arm32': dockerPush('arm32', version),
    'docker:push:arm64:latest': dockerPush('arm64'),
    'docker:push:arm64': dockerPush('arm64', version),
};
```

Reply:
great idea, I change that

Reply:
I just tried the above described but when running any `npm` command, such as `npm run fix:lint` I always get errors, which seem to be based on the changes above. I expect the `dockerPush(...)` calls to be issue. As long as they are commented, it compiles.

I simply just added the following lines to the end of the `.madrun.js`

```
};

function dockerPush(type, version = 'latest') {
    return `docker push coderaiser/cloudcmd:${version}-${type}`;
}

module.exports = {
    'docker:push:arm32': () => dockerPush('arm32', version),
    'docker:push:arm32:latest': () => dockerPush('arm32'),
    'docker:push:arm64': () => dockerPush('arm64', version),
    'docker:push:arm64:latest': () => dockerPush('arm64'),
};
```

Maybe you can see the error in there?

Reply:
What kind of an error do you have?

Reply:
```
Bad response: 422 {"message":"Couldn't find a repository matching this job.","error":true}
Command failed: nyc report --reporter=text-lcov | coveralls
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! cloudcmd@14.2.1 report: `madrun report`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the cloudcmd@14.2.1 report script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/$USER/.npm/_logs/2020-01-09T20_53_31_045Z-debug.log
```

It seems really a compile error in the `.madrun.js`

Reply:
I pulled your branch and can't reproduce this, in what directory are you located? do you have a script ` fix:lint` in your `package.json`? Anyways lint on [ci](https://travis-ci.org/coderaiser/cloudcmd/jobs/634946742) is passing :).

Reply:
ah okay, thats weird! But even cooler that it works in the `CI`. Seems that my local env is mixed up. I will check the `package.json` then

Reply:
Looks like you tried to overwrite `module.exports` in the bottom of a file.

Reply:
okay you were right, this seems to work now for me too 👍